lib/models/addons-factory.js

```diff
@@ -59,7 +59,7 @@ class AddonsFactory {
         } catch (e) {
           project.ui.writeError(e);
           const SilentError = require('silent-error');
-          throw new SilentError(`An error occured in the constructor for ${addonInfo.name} at ${addonInfo.path}`);
+          throw new SilentError(`An error occurred in the constructor for ${addonInfo.name} at ${addonInfo.path}`);
```

Comment:
this could theoretically break any script stupid enough to depend on the exact text of an error message.